lib/sqli/sqli_parser.y

```diff
@@ -360,7 +360,7 @@ alias_opt:
 table_ref:
           func alias_opt
         | colref_exact alias_opt
-        | '('[u1] select ')'[u2] alias_opt {YYUSE($u1); YYUSE($u2);}
+        | select_parens alias_opt
```

Comment:
Why did you change that? Does a table reference support SELECT without parentheses?

Reply:
This is a mistake. Thank you very much!